### ISSUE TITLE
External link is not preserved

### ISSUE THREAD

**Issue by Barthelemy** [htmlpreview/htmlpreview.github.com#43]
I use htmlpreview for the following page : https://github.com/AliceO2Group/CodingGuidelines/blob/master/coding_guidelines.html

If you "toggle all summaries" at the top and search for "Getting" you will find the following line : 

> For more details: Getting #includes right. 

The link should point to https://community.kde.org/Policies/Library_Code_Policy#Getting_.23includes_right with the following html : 

`<a href="https://community.kde.org/Policies/Library_Code_Policy#Getting_.23includes_right">Getting #includes right</a>.`

But with htmlpreview it points to https://htmlpreview.github.io/?https://github.com/AliceO2Group/CodingGuidelines/blob/master/coding_guidelines.html#Getting_.23includes_right

How to preserver this external link ? 

Thank you
Barth

**Comment by Barthelemy** [comment]
Any news ?

**Comment by TiffanyPender** [comment]
I agree, this is problematic. All external links are replaced with the 'htmlpreview' url to your repo. I hope a solution is reached soon.
